lib/email/utils/helpers.js

```diff
@@ -28,7 +30,7 @@ function logFlowEventFromMessage(log, message, type) {
   const elapsedTime = currentTime - flowBeginTime
 
   if (flowId && flowBeginTime && (elapsedTime > 0) && type && templateName) {
-    const eventName = `email.${templateName}.${type}`
+    const eventName = `email.${templateName}.${type}.${emailDomain}`
```

Comment:
> ...what if we didn't use flow events for this?

Sorry for the delayed response, I like this! I definitely don't think it's totally bananas or even half a banana. I reckon there's scope for refactoring the import scripts so that we can re-use some of the logic too, which reduces the implementation/infrastructure burden a little bit.

Question: Would some events be both flow events and email events? Or would we keep them separate and then `JOIN` between the two data sets?